beta-src/src/components/ui/units/WDFleetIcon.tsx

```diff
@@ -73,7 +73,23 @@ const WDFleetIcon: React.FC<IconProps> = function ({
           />
         </>
       )}
-      {iconState !== UIState.DISBANDED && (
+      {iconState === UIState.DESTROY && (
+        <>
+          <path
+            d="M15.3646 28.6305L12.3266 31.3526L12.5677 27.3504L2.21513 32.0883L9.94524 22.1305L7.18152 20.4889L9.59198 18.9803L0 10.0408L11.8695 14.0098L11.1837 1.79122L15.8259 9.62939L16.8026 4.80846L19.5372 11.8695L31.6768 0L23.9716 14.9241L26.7104 15.0986L25.2641 16.8275L36.3356 19.6744L25.7379 22.2178L28.2357 24.6408H24.5784L28.9754 31.9179L20.1398 27.4419L16.1459 34L15.3646 28.6305Z"
+            fill="#684C41"
+          />
+          <path
+            d="M16.4707 25.5757L14.0852 27.716L14.2556 24.8858L7.26941 28.0859L12.2275 21.6982L10.1329 20.4555L12.107 19.2212L6.22626 13.7395L13.5325 16.1832L13.0711 7.99178L16.5372 13.8434L17.2645 10.2568L18.9477 14.5956L25.9505 7.74658L21.5285 16.3079L23.5857 16.4367L22.526 17.7084L29.9693 19.6243L22.6673 21.374L24.4502 23.107H21.8569L24.8866 28.1233L19.5711 25.4302L17.0526 29.5613L16.4707 25.5757Z"
+            fill="#F1906E"
+          />
+          <path
+            d="M10.8814 25.2271L15.4613 23.1284L15.3407 25.1191L17.265 23.3944L17.7013 26.3908L19.1601 23.9928L21.9695 25.4183L19.9082 22.0104H21.7451L20.4734 20.7719L25.419 19.5875L20.569 18.3407L21.3586 17.3973L19.7835 17.2976L21.8573 13.2829L18.5242 16.5412L17.5974 14.1432L17.0447 16.857L14.4181 12.4226L14.7173 17.7381L10.6736 16.3874L13.8986 19.3922L12.2404 20.4312L13.8571 21.3912L10.8814 25.2271Z"
+            fill="#FEF189"
+          />
+        </>
+      )}
+      {iconState !== UIState.DISBANDED && iconState !== UIState.DESTROY && (
```

Comment:
Can you please briefly explain what is being done here?

Reply:
This is showing the destroy Icon SVG. I added another check on L92 to also check for UIState.DESTROY otherwise it was showing up like this with the destroy icon and the filled in star

![Screen Shot 2022-05-04 at 3 21 35 PM](https://user-images.githubusercontent.com/1848053/166835013-a8ec72b6-e84e-407e-aff2-c8e1304ba7f9.png)
s